test/debug_logger.ts

```diff
@@ -0,0 +1,10 @@
+import { DebugLogger } from '../src/contrib/debug';
+
+test('ConsoleLogger logs to the console', async (done) => {
```

Comment:
Do we gain anything from this test being async?

Reply:
We use await in the test

Reply:
:+1: